app/views/layouts/_application.html.erb

```diff
@@ -67,12 +67,24 @@
 
     <a href="#main-content" class="govuk-skip-link" tabindex="0">Skip to main content</a>
 
-    <%= govuk_header(service_name:, container_classes: ("govuk-width-container__wide" if wide_container_view?)) do |header|
+    <%= govuk_header(full_width_border: true, container_classes: ("govuk-width-container__wide" if wide_container_view?)) do |header|
       if user_signed_in?
         header.with_navigation_item(text: "Sign out", href: destroy_user_session_path)
       end
     end %>
 
+    <section aria-label="Service information" class="govuk-service-navigation" data-module="govuk-service-navigation">
```

Comment:
Is it worth using the govuk components as we do with `govuk_header` to avoid all the manual HTML?

https://govuk-components.netlify.app/components/service-navigation/

Reply:
Hi @ethax-ross I had tried that but due the conditional class I couldn't make it work. Had another go now and seems to work. Nice one. Tksss